src/Propel/Runtime/Util/PropelDateTime.php

```diff
@@ -76,16 +76,16 @@ public static function createHighPrecision($time = null)
     }
 
     /**
-     * Get the current microtime with milliseconds. Making sure that the decimal point separator is always "."
-     * and always including the fractional part. Otherwise self::createHighPrecision would return false.
+     * Get the current microtime with milliseconds. Making sure that the decimal point separator is always ".", ignoring
+     * what is set with the current locale. Otherwise self::createHighPrecision would return false.
      *
      * @return string
      */
     public static function getMicrotime()
     {
         $mtime = microtime(true);
 
-        return number_format($mtime, 6, '.', '');
+        return str_replace(',', '.', $mtime);
```

Comment:
@atompulse Sometimes now `PropelDateTime::createHighPrecision()` throws an error.
It happens, when the microtime has 0 micro seconds (e.g. 1496476204.0).
You convert the float value to string here (by `str_replace`). Result will be `'1496476204'`.
Then `$dateTime = \DateTime::createFromFormat('U.u', ...)` in line 71 will fail.